src/frontends/tensorflow_lite/src/decoder_flatbuffer.cpp

```diff
@@ -87,9 +92,38 @@ std::shared_ptr<ov::frontend::tensorflow_lite::TensorLitePlace> DecoderFlatBuffe
         ov::frontend::tensorflow_lite::get_ov_type(tensor->type()),
         names,
         ov::frontend::tensorflow_lite::get_quantization(tensor->quantization()),
-        tensor_info.input_idx,
-        tensor_info.output_idx,
-        (tensor_info.buffer->data() ? tensor_info.buffer->data()->data() : nullptr));
+        (tensor_info.buffer && tensor_info.buffer->data() ? tensor_info.buffer->data()->data() : nullptr));
+}
+
+ov::Any get_value_as_ov_any(const flexbuffers::Reference& value) {
+#define CASE_MACRO(fbt, as_stmt) \
+    case flexbuffers::fbt:       \
+        return {value.as_stmt()};
+    switch (value.GetType()) {
+        CASE_MACRO(FBT_INT, AsInt32)
+        CASE_MACRO(FBT_INDIRECT_INT, AsInt32)
+        CASE_MACRO(FBT_UINT, AsUInt32)
+        CASE_MACRO(FBT_INDIRECT_UINT, AsUInt32)
+        CASE_MACRO(FBT_FLOAT, AsFloat)
+        CASE_MACRO(FBT_INDIRECT_FLOAT, AsFloat)
+        CASE_MACRO(FBT_STRING, AsString)
+        CASE_MACRO(FBT_BOOL, AsBool)
+    default:
+        return {};
+    }
+    return {};
+}
+
+ov::Any DecoderFlatBuffer::get_attribute(const std::string& name) const {
+    const auto opts = m_node_def->custom_options();
+    if (opts == nullptr)
+        return {};
+    const flexbuffers::Map& m = flexbuffers::GetRoot(opts->Data(), opts->size()).AsMap();
+    try {
+        return get_value_as_ov_any(m[name]);
+    } catch (...) {
```

Comment:
I think you can catch only some specific types of exceptions....

Reply:
It appears the `flexbuffers::Map::operator[]` doesn't throw if it couldn't find the key with requested name -- it returns `flexbuffers::Reference` to a `nullptr` of `FBT_NULL` type which is being interpreted by `get_value_as_ov_any` with no problem.